src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -289,10 +289,14 @@ def make_dataframe_simulation_data(
         nb_real: number of realizations
 
     """
-    iteration = sorted(
-        [int(rel_iter.replace("/", "").split("-")[-1]) for rel_iter in glob.glob(path)]
-    )[-1]
-    runpath_list = glob.glob(path[::-1].replace("*", str(iteration)[::-1], 1)[::-1])
+    if "pred" in mode:
```

Comment:
So "historypred" is OK as well? Maybe just write it explicitly?

```python
if mode == "pred":
    # Something
elif mode == "ahm":
    # Something
else:
    raise ValueError("Not a valid mode... blabla")
```

In this case also update the docstring with "`Raises:`"